packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -66,61 +59,65 @@ const styles: ComponentStyles<ToggleClassNameContract, DesignSystem> = (
             borderRadius: "10px",
             width: "10px",
             height: "10px",
-            ...switchFieldPattern.rest.stateIndicator,
+            background: neutralForegroundRest,
+            "@media (-ms-high-contrast:active)": {
+                backgroundColor: "ButtonHighlight",
+            },
         },
         toggle_input: {
             position: "relative",
             margin: "0",
             width: "44px",
             height: "20px",
-            ...outlinePattern.rest,
+            background: neutralFillInputRest,
+            border: `${toPx(
+                designSystem.outlinePatternOutlineWidth
+            )} solid ${neutralOutlineRest(designSystem)}`,
             borderRadius: "20px",
             appearance: "none",
             outline: "none",
             "&:hover": {
-                ...outlinePattern.hover,
+                background: neutralFillInputHover,
+                border: `${toPx(
+                    designSystem.outlinePatternOutlineWidth
+                )} solid ${neutralOutlineHover(designSystem)}`,
+            },
+            "&:active": {
+                background: neutralFillInputActive,
+                border: `${toPx(
+                    designSystem.outlinePatternOutlineWidth
+                )} solid ${neutralOutlineActive(designSystem)}`,
             },
             ...applyFocusVisible({
-                ...outlinePattern.focus,
+                boxShadow: `0 0 0 1px ${designSystem.foregroundColor} inset`,
+                border: `${toPx(designSystem.outlinePatternOutlineWidth)} solid ${
+                    designSystem.foregroundColor
+                }`,
             }),
         },
-        toggle__disabled: {
-            ...typographyPattern.disabled,
-            "& $toggle_input, & $toggle_label, & $toggle_statusMessage": {
-                cursor: "not-allowed",
-            },
+        toggle__checked: {
             "& $toggle_input": {
-                ...outlinePattern.disabled,
-            },
-            "& $toggle_stateIndicator": {
-                ...switchFieldPattern.disabled.stateIndicator,
-                "@media screen and (-ms-high-contrast:active)": {
-                    background: "ButtonText",
-                },
-            },
-            "&$toggle__checked": {
-                "& $toggle_input": {
-                    ...outlinePattern.disabled,
-                    borderColor: "transparent",
-                    ...switchFieldPattern.disabled.stateIndicator,
-                },
-                "& $toggle_stateIndicator": {
-                    background: backgroundColor,
+                background: accentFillRest,
+                borderColor: accentFillRest,
+                "& + $toggle_stateIndicator": {

```

Comment:
Do you actually need to complicate this selector with a sibling selector? Did you try just selecting for `$toggle__checked $toggle_stateIndicator` ?